CHANGES.md

```diff
@@ -1,5 +1,97 @@
 # GAP - history of changes
 
+## GAP 4.12.1 (October 2022)
+
+### Performance improvements
+
+- [#5032](https://github.com/gap-system/gap/pull/5032) Fix performance regression in `AllHomomorphismClasses` and possibly other code searching isomorphisms
+
+### Improved and extended functionality
+
+- [#5047](https://github.com/gap-system/gap/pull/5047) `Display` for a character table optionally shows the degrees of the character fields
+
+### Fixed bugs that could lead to incorrect results
+
+- [#5036](https://github.com/gap-system/gap/pull/5036) Fix `Centre` for pc groups giving wrong results, e.g. `Size(Center(SmallGroup(2^9,261648)))` gave 4 instead of 8
+- [#5067](https://github.com/gap-system/gap/pull/5067) Fix `AbsolutelyIrreducibleModules` for a pc group and a finite prime field
+
+### Fixed bugs that could lead to crashes
+
+- [#5126](https://github.com/gap-system/gap/pull/5126) Fix crashes in `ConvertToVectorRep` and `CopyToVectorRep`
+
+### Fixed bugs that could lead to break loops
+
+- [#5025](https://github.com/gap-system/gap/pull/5025) Fix unexpected error in tab completion for non-record component objects
+- [#5033](https://github.com/gap-system/gap/pull/5033) Fix unexpected error in `ConjugacyClassesSubgroups`
+- [#5041](https://github.com/gap-system/gap/pull/5041) Fix unexpected error in `MinimalGeneratingSet` for solvable non-pc groups
+- [#5057](https://github.com/gap-system/gap/pull/5057) Fix `Orbits` when called with a range as seeds
+
+### Other fixed bugs
+
+- [#5043](https://github.com/gap-system/gap/pull/5043) Fix `SaveOnExitFile` in restored workspaces
+- [#5116](https://github.com/gap-system/gap/pull/5116) Fix printing of GAP functions containing sublist extractions
+
+### Other changes
+
+- [#5079](https://github.com/gap-system/gap/pull/5079) Improve buildsystem by exporting `GMP_PREFIX` in `sysinfo.gap` for package build systems
+
+### Package distribution
+
+#### Updated packages redistributed with GAP
+
+The GAP 4.12.0 distribution contains 154
+packages, of which 47 have been updated since GAP
+4.11.1. The full list of updated packages is given below:
```

Comment:
It should show what GAP 4.12.1 contains as well as the updates since GAP 4.12.0.

Reply:
Thanks, fixed